include/verilated.h

```diff
@@ -2487,6 +2488,44 @@ static inline void VL_ASSIGNSEL_WIIW(int obits, int lsb, WDataOutP owp, WDataInP
     _VL_INSERT_WW(obits, owp, rwp, lsb + obits - 1, lsb);
 }
 
+//======================================================================
+// Variable randomization (std::randomize, randomize method)
+
+#define VL_DEF_RANDOMIZE_VAR(varType) \
+inline IData VL_RANDOMIZE_VAR(varType& odata, int obits) VL_MT_SAFE { \
+    odata = VL_RANDOM_Q(obits); \
+    return 1; \
+}
+VL_DEF_RANDOMIZE_VAR(CData)
+VL_DEF_RANDOMIZE_VAR(SData)
+VL_DEF_RANDOMIZE_VAR(IData)
+VL_DEF_RANDOMIZE_VAR(QData)
```

Comment:
I'd probably just replicate the code as only 8 more lines in total and more obvious.  Note C/S/I should presumably call VL_RANDOM_I not _Q.